charts/jenkins/templates/tests/jenkins-test.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   {{- end }}
   initContainers:
     - name: "test-framework"
-      image: "bats/bats:1.2.1"
+      image: {{ .Values.controller.image }}:{{ .Chart.AppVersion }}-{{ .Values.controller.tagLabel }}
```

Comment:
bats is not the controller image, it's a test framework.

https://github.com/jenkinsci/helm-charts/issues/683 says it should be written in the same style, not to use it literally

Reply:
Hi @timja I am new just finding my way about open source and Jenkins.
I reviewed https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/templates/jenkins-controller-statefulset.yaml
And figured initContainer images uses the below so I'll edit and pull again
image: "{{ .Values.controller.image }}:{{- include "controller.tag" . -}}"